src/script/components/MessagesList/index.tsx

```diff
@@ -232,7 +233,7 @@ const MessagesList: React.FC<MessagesListParams> = ({
   });
 
   return (
-    <div ref={setContainer} className={`messages ${verticallyCenterMessage() ? 'flex-center' : ''}`}>
+    <div ref={messagesContainerElement} className={`messages ${verticallyCenterMessage() ? 'flex-center' : ''}`}>
```

Comment:
not a big fan of classnames because libs like cslx are more versatile and if I just need a class string, string templates are a very good option.

But since we have the lib in use anyways....